pkg/kube/inject/inject.go

```diff
@@ -1100,6 +1100,28 @@ func potentialPodName(metadata *metav1.ObjectMeta) string {
 	return ""
 }
 
+
+// rewriteCniPodSpec will check if values from the sidecar injector Helm
+// values need to be inserted as Pod annotations so the CNI will apply
+// the proper redirection rules.
+func rewriteCniPodSpec(annotations map[string]string, spec *SidecarInjectionSpec) {
```

Comment:
Oops. I had purposefully skipped this stuff, cause I think it's fixed upstream, but part of it may have only made it into the 1.9 release.